src/base/Context.ts

```diff
@@ -1158,12 +1155,14 @@ export class Context {
     return cloned;
   }
 
+  /* eslint-disable @typescript-eslint/no-explicit-any */
   /**
    *  @hidden
    *
    * Creates an instance of a type as registered in the polymeshApi instance
    */
-  public createType<K extends keyof InterfaceTypes>(type: K, params: unknown): InterfaceTypes[K] {
+  public createType<K extends keyof InterfaceTypes>(type: K, params: unknown): any {
```

Comment:
They made a change that was supposed to make the createType API smarter, but now `InterfaceTypes[K]` exceeds TypeScript stack size, and might be infinite recursion even.